OpenRA.Mods.Common/Traits/World/PathFinder.cs

```diff
@@ -73,7 +75,7 @@ public void WorldLoaded(World w, WorldRenderer wr)
 			if (sourcesList.Count == 0)
 				return NoPath;
 
-			var locomotor = GetLocomotor(self);
+			var locomotor = GetUnitLocomotor(self);
```

Comment:
Can we please keep this as `GetLocomotor` (or otherwise use `GetActorLocomotor`)? Our current naming conventions try to avoid terms like "Unit" since all we really care about are traits.

Reply:
Changed to `GetActorLocomotor`.

On a similar theme, should `IPathFinder.FindUnitPathToTargetCell` and `IPathFinder.FindUnitPathToTargetCellByPredicate` really be `FindActorPath...` instead?

Reply:
Yeah, or even just `PathFinder.FindPathToTargetCellByPredicate`?